packages/core/core/src/types.ts

```diff
@@ -443,3 +443,11 @@ export interface AgnosticFacetSearchParams {
   metadata?: any;
   [x: string]: any;
 }
+
+export interface AgnosticLogger {
+  debug(message?: any, ...args: any): void;
+  info(message?: any, ...args: any): void;
+  warn(message?: any, ...args: any): void;
+  error(message?: any, ...args: any): void;
+  verbosity: string;
```

Comment:
you mean `defaultVerbosity` ? should that be even in this interface? imho its more for `registerLogger`